components/base/kinematicbase/differentialDrive.go

```diff
@@ -8,22 +8,23 @@ import (
 	"math"
 	"time"
 
+	"github.com/edaniels/golog"
 	"github.com/golang/geo/r3"
-	utils "go.viam.com/utils"
 
 	"go.viam.com/rdk/components/base"
 	"go.viam.com/rdk/motionplan"
 	"go.viam.com/rdk/referenceframe"
 	"go.viam.com/rdk/services/motion"
 	"go.viam.com/rdk/spatialmath"
+	utils "go.viam.com/utils"
 )
 
 const (
-	distThresholdMM         = 100
+	distThresholdMM         = 1000
```

Comment:
Can we add some comments here discussing what these are used for/what they represent? "distThresholdMM` and `deviationThreshold` look very similar.